.gitlab-ci-check-license.yml

```diff
@@ -0,0 +1,23 @@
+
+stages:
+  - test
+
+test:check-license:
+  image: alpine
+  stage: test
+  before_script:
+    # Install dependencies
+    - apk add --no-cache git bash perl-utils
+    # Rename the branch we're on, so that it's not in the way for the
+    # subsequent fetch. It's ok if this fails, it just means we're not on any
+    # branch.
+    - git branch -m temp-branch || true
+    # Git trick: Fetch directly into our local branches instead of remote
+    # branches.
+    - git fetch origin 'refs/heads/*:refs/heads/*'
+    # Get last remaining tags, if any.
+    - git fetch --tags origin
+    - git clone http://github.com/mendersoftware/mendertesting /tmp/mendertesting
+  script:
+    # Check licenses
+    - /tmp/mendertesting/check_license.sh
```

Comment:
Both this test and the `check_commits.sh` are duplicated, since they are already running in the unit test stage. Is the intention here that this be reusable downstream?

Reply:
Bah, never mind, you already answered it. Sorry, I wasn't part of the original review. Moving on...